lib/picam/camera.ex

```diff
@@ -16,11 +16,18 @@ defmodule Picam.Camera do
     port =
       Port.open({:spawn_executable, executable}, [{:packet, 4}, :use_stdio, :binary, :exit_status])
 
-    {:ok, %{port: port, requests: []}}
+    offline_image = image_data("offline_1280_720.jpg")
```

Comment:
I love the semantics of `image_data` here.  Very intuitive function name.  I'd like to provide the image filename as a parameter in `opts` which defaults to the image filename you've provided.    I'd do this using `Keyword.pop` or `Keyword.get`:

```elixir
def init(_opts) do
  port = spawn_port()

  offline_image = Keyword.get(opts, :offline_image, "offline_1280_720.jpg") |> image_data()

  {:ok, %{port: port, requests: [], offline: false, offline_image: offline_image}}
end
```

Or something like that.

Reply:
yeah good idea